cms/djangoapps/contentstore/management/commands/backfill_orgs_and_org_courses.py

```diff
@@ -78,20 +78,20 @@ def handle(self, *args, **options):
             # function more deterministic in case something goes wrong.
             for orgslug in sorted(orgslugs)
         ]
-        org_coursekey_pairs = [
-            ({"short_name": orgslug}, coursekey)
-            for orgslug, coursekey in sorted(orgslug_coursekey_pairs)
+        org_courseid_pairs = [
+            ({"short_name": orgslug}, courseid)
+            for orgslug, courseid in sorted(orgslug_courseid_pairs)
```

Comment:
This line is where the exception occurred. We're sorting the org/course_key pairs to make it so that order of the backfill isn't dependent on the state of the CourseOverview table.

I thought that CourseKeys were sortable. It turns out that only certain subsets of CourseKeys (for example, all Split Mongo course locators) are sortable.